metrixpp/tests/general/test_basic/sources/simple.cpp

```diff
@@ -64,6 +64,9 @@ class A
 		return 1;
 	}
 
+    virtual int pure_virtual_method() = 0;

```

Comment:
Have not reviewed the tests. I hope you invoke the plugin over this code? If yes, I am happy to merge. Thanks for contribution.